src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -151,13 +152,15 @@ import Network.HTTP.Req.Extra (httpConfigRetryTimeouts)
 import Network.HTTP.Types qualified as HTTP
 import Parse.XML (FromXML (..), child, parseXML, xmlErrorPretty)
 import Path (File, Path, Rel)
+import Path qualified
```

Comment:
[minor] we should do the explicit import for toFilePath

Reply:
As in, instead of `Path.toFilePath` you think we should `import Path (File, Path, Rel, toFilePath)`?

Reply:
Went ahead and made this change but if you meant something else let me know